TLM/TLM/UI/WhatsNew/WhatsNew.cs

```diff
@@ -11,7 +11,7 @@ namespace TrafficManager.UI.WhatsNew {
 
     public class WhatsNew {
         // bump and update what's new changelogs when new features added
-        internal static readonly Version CurrentVersion = new Version(11, 6, 4, 0);
+        internal static readonly Version CurrentVersion = new Version(11,6,4,1);
 
         internal static readonly Version PreviouslySeenVersion = GlobalConfig.Instance.Main.LastWhatsNewPanelVersion;
```

Comment:
This cannot be readonly because MarkAsShown won't work properly

Reply:
Wait, it's always been that way - here's the original version of the class: https://github.com/CitiesSkylinesMods/TMPE/commit/7cd4a305e781f43a3b1057c9ae223076ce4cd1b4#diff-fa2814c9d64acf4ea7741ee6d806ae2e559b946734217b0c5ad1f0d50566385eR22

Also, why won't MarkAsShown() work? It only updates global config, and the value of that is accessed via `PreviouslySeenVersion` (which could arguably be a getter/setter tbh)

When global config is loaded, is it trying to update something in `WhatsNew.cs`?

Reply:
Maybe it's the `Shown` getter that was causing problem. I assume you were seeing it every time when opening TM:PE panel?

will send commit shortly

Reply:
I'm talking about `PreviouslySeenVersion` -> once mod is initialized it will never change its value so even if you call `MarkAsShown()`. Yes, to check if we should show we use `Shown`, but now it returns always the same value no matter if you  haven't seen it and the later opened

Reply:
@krzychu124 I've just pushed a commit to this PR that should fix it - haven't had time to test it in-game myself yet (will be another 30-45 mins before I get chance to do that)

Reply:
Testing...